cmd/clusterctl/client/cluster/objectgraph_test.go

```diff
@@ -1303,9 +1277,7 @@ func Test_objectGraph_setClusterTenants(t *testing.T) {
 				sort.Strings(wantTenants)
 				sort.Strings(gotTenants)
 
-				if !reflect.DeepEqual(gotTenants, wantTenants) {
-					t.Fatalf("cluster %s, got = %s, expected = %s", cluster.identity.UID, gotTenants, wantTenants)
-				}
+				g.Expect(gotTenants).To(Equal(wantTenants))
```

Comment:
```suggestion
				g.Expect(gotTenants).To(ConsistOf(wantTenants))
```
Also can remove the `sort.Strings()` above.